### DISCUSSION TITLE
Onnx input size

### DISCUSSION THREAD

**Discussion by AD-HO** [onnx/onnx#2977] [Q&A]
Hi Everyone,
I would like to know how to unfreeze the input size of an exported Onnx model?
Example: https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html

Super-Resolution is an image-to-image translation, i can’t see how i would consume this model without resizing the input to 224 x 224 , and this mean the input quality will decrease
thank you.

**Comment by ytaous** [comment]
please see - https://github.com/microsoft/onnxruntime/issues/4929

**Comment by yywbxgl** [comment]
you can convert the input size to Dynamic input like ( 0 ,3 ,224, 224) ,  Then  the onnxruntime can accept  diffrent batch images as input.  （1，3，0, 0） mean you can can input image by  different size.  But the prerequisite for this is that the model cab adapt to dynamic input，For example, there is no operator with a fixed shape like reshape。
